lib/bonny/operator.ex

```diff
@@ -145,18 +145,31 @@ defmodule Bonny.Operator do
   @doc false
   @spec run({atom(), Bonny.Resource.t()}, {module(), keyword()}, module(), K8s.Conn.t()) :: :ok
   def run({action, resource}, controller, operator, conn) do
-    Axn.new!(
-      conn: conn,
-      action: action,
-      resource: resource,
+    metadata = %{
+      operator: operator,
       controller: controller,
-      operator: operator
-    )
-    |> operator.call([])
-    |> Bonny.Axn.emit_events()
-    |> Bonny.Axn.run_after_processed()
-
-    :ok
+      action: action,
+      name: K8s.Resource.name(resource),
+      namespace: K8s.Resource.namespace(resource),
+      kind: K8s.Resource.kind(resource),
+      api_version: resource["apiVersion"],
+      library: :bonny
+    }
+
+    :telemetry.span([:operator, :run], metadata, fn ->
```

Comment:
An alternative would be to have the action as part of event it.

eg: `[:operator, :reconcile, :start]` or `[:operator, :delete, :exception]`

But I kept it as run to be in line with the previous ones.

Reply:
I think valid approaches. This way consumers can still filter through the metadata.

Reply:
Yep, maybe in the future the first atom should include "bonny" as to avoid possible clashes, but that's out of scope of this PR.